_posts/en/newsletters/2019-12-18-newsletter.md

```diff
@@ -0,0 +1,238 @@
+  all be claimed at the same time by the receiver.  This provides a
+  major upgrade to the usability of LN by allowing users with their
+  balance split across multiple channels to spend or receive funds in
+  multiple channels in the same overall payment.  With this upgrade,
+  normal non-routing users don't need to worry about how much balance or
+  capacity they have in any particular channel---they can spend or
+  receive up to their full available balance or capacity (subject to
+  other limits of their software or the protocol).
+
+- **Analysis of bech32 error detection:** Pieter Wuille sent an
+  [email][wuille bech32 post] to the Bitcoin-Dev mailing list following
+  up on the bech32 malleability concerns described in previous
+  newsletters ([#72][news72 bech32], [#74][news74 bech32], and
+  [#76][news76 bech32]) where it's possible to add or remove a `q`
+  character immediately before a `p` character at the end of a bech32
+  string.  Wuille's [analysis][wuille bech32 analysis] shows that this
+  is the only exception to bech32's expected error detection properties
+  and that "changing one constant in bech32 would resolve this issue."
+
+    Wuille plans to amend [BIP173][] to describe the weakness, propose a
+    change to limit existing bech32 address uses to either 20 byte or 32
+    byte witness program payloads, and define a modified version of
+    bech32 with the alternative constant for non-Bitcoin uses and for a
+    potential future where we want witness programs that are not 20
+    bytes or 32 bytes.
+
+- **Proposed changes to bip-ctv:** Jeremy Rubin [suggested][rubin ctv
+  update] several changes to the proposed soft fork addition of an
+  `OP_CHECKTEMPLATEVERIFY` ([CTV][topic covenants]) opcode.  Most
+  notably, the changes will do away with the restriction that template
+  used by CTV cannot be derived from other data via Bitcoin Script.
+  This change simplifies the change to the Script language as discussed
+  in [Newsletter #75][news75 ctv].  Neither this change nor any of the
+  other changes modifies CTV's behavior in a way that would
+  significantly affect previously-described usecases as far as we're
+  aware (but anyone who is aware of fundamental changes is encouraged to
+  discuss them on-list).
+
+- **Discussion of eclipse attacks on LN nodes:** Antoine Riard
+  [posted][riard eclipse] to the Lightning-Dev mailing list a
+  description of two attacks possible against LN users if they are
+  eclipse attacked and the attacker delays the relay of blocks.  An
+  eclipse attack occurs when all connections made by a full node or
+  lightweight client are controlled by a single attacker, as might
+  easily be the case when the attacker is an ISP or hacks into a user's
+  router.  This gives the attacker full control over what data the node
+  or client sends or receives.  In the first attack, the eclipse
+  attacker can confirm a revoked commitment transaction without the
```

Comment:
Suggestion:
The end effect is that the attacker 'can confirm a revoked commitment transaction', but isnt the attacker simply blocking awareness of the revoked commitment transaction's broadcast/confirmation?

If so, my suggestion would be attacker blocking verbiage instead of attacker confirming.